### ISSUE TITLE
Add metric: security policy 

### ISSUE THREAD

**Issue by ljharb** [ljharb/repo-report#73]
This should respect a .github repo (org or user), as well as a repo-specific SECURITY.md.

**Comment by abdumamdouh** [comment]
i will work on this

**Comment by khattakdev** [comment]
Hey @ljharb 
I went through the code with @abdumamdouh. 
I noticed that this metric is already added
```
SecurityPolicyEnabled: {
    extract: (item) => item.isSecurityPolicyEnabled,
    permissions: ["ADMIN", "MAINTAIN", "WRITE"],
  },
 ```
 This is from `config/metrics.js`
 
 However, the issue is it's returning the opposite of what it should return.
 
 It returns false for the `repo-report` and `repo-report-cache` which have security police enabled.
 I tested it on my repositories and it returns false for those which have a security policy but returns true for the one that doesn't have any security policy.
 
 Any hints/ideas on why this is happening?

![image](https://user-images.githubusercontent.com/37709578/159993070-734a0d90-a81d-4220-9676-e4119b911eb9.png)

`dummy-1` has a security policy but it's printing false and true for other repos where I don't have a security policy.

**Comment by ljharb** [comment]
hmm. That strongly suggests that `SecurityPolicyEnabled` isn't what we think it is. There's a bunch of things in the API that are confusing like "disabled", and https://docs.github.com/en/rest/reference/repos doesn't reference `isSecurityPolicyEnabled` at all.

If you delete the security policy file from `dummy-1`, does it immediately switch back to `true`? If you then add it under `khattakdev/.github`, does it switch back to `false`? Maybe their boolean logic is just flipped, and we just need to add a `!`?

**Comment by khattakdev** [comment]
@ljharb tried what you said. I removed `SECURITY.md` from a repo and the API returns true.

So I guess we have to reverse it.

**Comment by ljharb** [comment]
Sounds good, let's do that then :-) let's please add a comment tho, and link it to https://github.com/github/feedback/discussions/13705 which i just filed?

**Comment by ljharb** [comment]
Closed in #75.
